### ISSUE TITLE
Asterisk in \id line causes failure of \h line

### ISSUE THREAD

**Issue by philleckrone** [sillsdev/ptx2pdf#796]
The text I was trying to work with had the following \id line:
`\id GAL - GPS Training New Testament - ***for training only - not for publication***`

When this is run into PTXPrint the following \h line is added to the end of the \id and "lost"
When the PDF is created there is no book on the header.
Removing the asterisks fixes the issue, but I'm not sure most people would know to look for this.

**Comment by davidg-sil** [comment]
When my input looks like this:
```
\id JHN -  *** Test version ***
\h John
```
the pre-processed form (from the latest pre-build which we're just in the process of unbreaking) looks like this:
```
\id JHN ‑ \asterisk \asterisk \asterisk\space Test version \asterisk \asterisk \asterisk\space \h John
```
Since the new line at the end of the ID line is significant to the TeX code, 
(it ignores all but the first 3 letters of the ID line) the `\h`  getting lost is thus explained.
Adding a letter after the final asterisk returns behaviour to normal.

**Comment by markpenny** [comment]
Will be fixed in 2.3.10
